config/manager/manager.yaml

```diff
@@ -28,7 +28,11 @@ spec:
     spec:
       containers:
       - args:
-        - --enable-leader-election
+        - "--metrics-addr=127.0.0.1:8080"
+        - "--enable-leader-election"
+        - "--zap-log-level=info"
+        - "--zap-time-encoding=rfc3339nano"
+        - "--feature-gates=+operator.autoinstrumentation.go,+operator.autoinstrumentation.nginx"
```

Comment:
I moved this so I could patch these args in the manager kustomization.yaml, as opposed to the one in config/default. I don't see any reason for it not to be here.